tests/vmi_multus_test.go

```diff
@@ -63,7 +63,7 @@ const (
 const (
 	sriovnet1 = "sriov"
 	sriovnet2 = "sriov2"
-	sriovnet3 = "sriov-link-enabled"
+	sriovnet3 = "sriov3"
```

Comment:
Please change sriovnet3 variable *name* to `sriov-link-enabled` to represent its unique feature

Reply:
This has been already discussed/raised multiple time in the review. The information is not of interest and this whole sriov stuff needs to be refactored.

Reply:
If I undersntand correctly you created those variables so it will be easier to understand what goes where 
On this [line](https://github.com/kubevirt/kubevirt/blob/3ef507d06cebeb0bc782c2f5c03bab3f9e9731d4/tests/vmi_multus_test.go#L846) where we actually use it, you cant figure out from sriovnet3 that we going to connect the VM's to sriov-network with link-enable.

Reply:
1. If one wants to know, he can follow the source like with the others.
2. This link-enable needs either to be dropped or set to all others. I think it can be dropped.